src/modules/deal-investor/deal-investor.controller.test.ts

```diff
@@ -0,0 +1,65 @@
+import { DealInvestorGenerator } from '@ukef-test/support/generator/deal-investor-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { when } from 'jest-when';
+
+import { DealInvestorController } from './deal-investor.controller';
+import { DealInvestorService } from './deal-investor.service';
+
+jest.mock('./deal-investor.service');
+
+describe('DealInvestorController', () => {
+  const valueGenerator = new RandomValueGenerator();
+
+  let controller: DealInvestorController;
+  let dealInvestorService: DealInvestorService;
+
+  let dealInvestorServiceGetDealInvestors: jest.Mock;
+
+  beforeEach(() => {
+    dealInvestorService = new DealInvestorService(null, null);
+
+    dealInvestorServiceGetDealInvestors = jest.fn();
+    dealInvestorService.getDealInvestors = dealInvestorServiceGetDealInvestors;
+
+    controller = new DealInvestorController(dealInvestorService);
+  });
+
+  describe('getDealInvestors', () => {
+    const dealIdentifier = valueGenerator.ukefId();
+    const portfolioIdentifier = valueGenerator.string();
+
+    const { dealInvestors, dealInvestorsFromApi } = new DealInvestorGenerator(valueGenerator).generate({
```

Comment:
See comment on `deal-investor-generator.ts`, could this be changed to `const { dealInvestorsFromService, dealInvestorsFromApi }`